api/compression.md

```diff
@@ -13,12 +13,32 @@ manually](/compression/compress_chunk), instead of automatically as they age.
 
 ### Restrictions
 
-The current version does not support altering or inserting data into compressed
-chunks. The data can be queried without any modifications, however if you
-need to backfill or update data in a compressed chunk you will need to
+The current version does not fully support altering or inserting data into
+compressed chunks. The data can be queried without any modifications, however
+if you need to backfill or update data in a compressed chunk you will need to
 decompress the chunk(s) first.
 
 Starting with TimescaleDB 2.1, users have the ability to modify the schema
 of hypertables that have compressed chunks.
 Specifically, you can add columns to and rename existing columns of 
-such compressed hypertables.
\ No newline at end of file
+such compressed hypertables.
+
+Starting with TimescaleDB 2.3, users have the ability to insert data into
+compressed chunks, and enable compression policies on distributed hypertables.
+
```

Comment:
I would rewrite this part a bit.  The general restrictions and limitations around inserts are combined together here.
For 2.3. section (this paragraph) , I would just add this blob and move all the other restrictions (about type of constraints and row level security) further up as these haven't been supported since day 1.
```
There are however some limitations to inserting into compressed chunks. Specifically,
- upserts are not supported
- RETURNING clause is not supported
-  After Insert row triggers are not supported.
-  Inserts into chunks with primary key or unique constraints is not supported.
```